bgpd/bgp_main.c

```diff
@@ -469,6 +476,23 @@ main (int argc, char **argv)
 	  if (vty_port <= 0 || vty_port > 0xffff)
 	    vty_port = BGP_VTY_PORT;
 	  break;
+	case OPTION_VTYSOCK:
+	  vty_sock_name = strrchr (BGP_VTYSH_PATH, '/');
```

Comment:
In the process of updating. Will create a new pull request to keep git history clean. Leave this pull req open until I have the new one submitted, afterwards it can be deleted.